db/blob/blob_source_test.cc

```diff
@@ -108,33 +109,34 @@ class BlobSourceTest : public DBTestBase {
  protected:
  public:
   explicit BlobSourceTest()
-      : DBTestBase("blob_source_test", /*env_do_fsync=*/true) {}
+      : DBTestBase("blob_source_test", /*env_do_fsync=*/true) {
```

Comment:
https://github.com/facebook/rocksdb/pull/10248/files#diff-6f21588c74bc9b73a3f493589078acd9e29bd850f078899d3df7419585ce435cL82. The cause of `zstd` compression failing is unknown to me. Any compatibility problems there?

Reply:
It works pretty well on my server. I'll try using snappy instead.

Reply:
Yeah, just use Snappy - we also use the helper `Snappy_Supported()`  to bail out if it's not available. We do this in plenty of tests

Reply:
done